homeassistant/components/withings/coordinator.py

```diff
@@ -37,11 +37,15 @@ class WithingsDataUpdateCoordinator(DataUpdateCoordinator[_T]):
     _default_update_interval: timedelta | None = UPDATE_INTERVAL
     _last_valid_update: datetime | None = None
     webhooks_connected: bool = False
+    coordinator_name: str = ""
```

Comment:
I'd make this an `abstractmethod` property instead that must be implemented by child classes. We never want it to be used when it's an empty string so this gives an incorrect impression.